### DISCUSSION TITLE
I get a 404 page in vercel after adding next-i18next

### DISCUSSION THREAD

**Discussion by ryosuke-hujisawa** [vercel/next.js#28884] [Help]
### What version of Next.js are you using?

10.0.0

### What version of Node.js are you using?

v12.13.0

### What browser are you using?

Chrome

### What operating system are you using?

mac

### How are you deploying your application?

Vercel

### Describe the Bug

I get a 404 page in vercel after adding next-i18next

Here are the details of my bug.

https://stackoverflow.com/questions/69088768/i-get-a-404-page-in-vercel-after-adding-next-i18next

### Expected Behavior

The correct page is displayed instead of a 404 page

### To Reproduce

https://stackoverflow.com/questions/69088768/i-get-a-404-page-in-vercel-after-adding-next-i18next

**Comment by stefanprobst** [comment]
please provide steps how others can reproduce this error, or a link to the source repo, otherwise it's difficult to help.
